functions/Remove-DbaDbBackupRestoreHistory.ps1

```diff
@@ -54,75 +54,69 @@ function Remove-DbaDbBackupRestoreHistory {
     .EXAMPLE
         PS C:\> Remove-DbaDbBackupRestoreHistory -SqlInstance sql2016
 
-        Remove backup and restore history on SQL Server sql2016 older than 30 days (default period)
+        Prompts for confirmation then deletes backup and restore history on SQL Server sql2016 older than 30 days (default period)
 
-        PS C:\> Remove-DbaDbBackupRestoreHistory -SqlInstance sql2016 -KeepDays 100
+        PS C:\> Remove-DbaDbBackupRestoreHistory -SqlInstance sql2016 -KeepDays 100 -Confirm:$false
 
-        Remove backup and restore history on SQL Server sql2016 older than 100 days
+        Remove backup and restore history on SQL Server sql2016 older than 100 days. Does not prompt for confirmation.
 
         PS C:\> Remove-DbaDbBackupRestoreHistory -SqlInstance sql2016 -Database db1
 
-        Remove complete backup and restore history for database db1 on SQL Server sql2016 
+        Prompts for confirmation then deletes all backup and restore history for database db1 on SQL Server sql2016
 
         PS C:\> Get-DbaDatabase -SqlInstance sql2016 | Remove-DbaDbBackupRestoreHistory -WhatIf
 
-        Remove complete backup and restore history for all databases on SQL Server sql2016 
-#>
-    [CmdletBinding(SupportsShouldProcess)]
+        Remove complete backup and restore history for all databases on SQL Server sql2016
+    #>
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'High')]
```

Comment:
Changed the confirm impact to high to prompt by default, then added help to show people how to shush it.